### PR TITLE
DOCS: Add example of visualizing JSON profile on Chromium

### LINKED ISSUE #12212: analyze-profile: Docs are out of date? `--html`, `--dump`, etc. are not in `help --long`?
### Description of the problem / feature request:

Docs seem stale; seems like it documents the old profiling setup (ye olde profile file), not the current one (JSON).
Also, `--dump` and `--html` do not seem to work as indicated on the docs.

### Feature requests: what underlying problem are you trying to solve with this feature?

Just profiling build, namely to get test timing info w/o parsing text / JSON / whatevs.

### Bugs: what's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

See docs here:
https://docs.bazel.build/versions/3.5.0/user-manual.html#analyze-profile
https://docs.bazel.build/versions/3.5.0/guide.html#profiling

### What operating system are you running Bazel on?

Ubuntu 18.04

### What's the output of `bazel info release`?

release 3.5.0

### If `bazel info release` returns "development version" or "(@non-git)", tell us how you built Bazel.

N/A

### What's the output of `git remote get-url origin ; git rev-parse master ; git rev-parse HEAD` ?

N/A. See standalone repro script below, and its output.

###  Have you found anything relevant by searching the web?

* #10140
* Code doesn't smell like it matches the docs: https://github.com/bazelbuild/bazel/blob/3.5.0/src/main/java/com/google/devtools/build/lib/runtime/commands/ProfileCommand.java

### Any other information, logs, or outputs that you want to share?

See standalone reproduction script and output:
https://github.com/EricCousineau-TRI/repro/blob/9cc40aa7bd7d0e3370afe0e0cef41c6daacb96c5/bazel/bazel_3.5.0_profile_example/repro.sh
https://github.com/EricCousineau-TRI/repro/blob/9cc40aa7bd7d0e3370afe0e0cef41c6daacb96c5/bazel/bazel_3.5.0_profile_example/repro.output.txt

### PR DESCRIPTION
**Pull request by EricCousineau-TRI**
Remove --dump=raw, since it isn't really supported

Resolves #12212

### CODE REVIEW

File: site/docs/guide.md [hunk 1]
```diff
@@ -1082,9 +1082,38 @@ The first step in analyzing the performance of your build is to profile your
...
 
 The file generated by the [`--profile`](user-manual.html#flag--profile) flag is
-a binary file. Once you have generated this binary profile, you can analyze it
-using Bazel's [`analyze-profile`](user-manual.html#analyze-profile') command. By
-default, it will print out summary analysis information for each of the
+a JSON file. Once you have generated this JSON profile, you can analyze it
+using Chrome's [`chrome://tracing`](https://dev.chromium.org/developers/how-tos/trace-event-profiling-tool) tool or Bazel's
+[`analyze-profile`](user-manual.html#analyze-profile') command.
+
+### Chrome's `chrome://tracing`
```
Comment (meisterT) [inline site/docs/guide.md:1089]: This is described here: https://docs.bazel.build/versions/master/skylark/performance.html#performance-profiling

We need to work on consolidating the docs on performance (see also https://github.com/bazelbuild/bazel/issues/11384)

Comment (EricCousineau-TRI) [reply site/docs/guide.md:1089]: Oohhh, thanks! Didn't search hard enough :sweat: At a minimum, I can convert this PR to at least cross-reference (unless I missed those too)

Comment (meisterT) [reply site/docs/guide.md:1089]: Yes, please do a cross reference.

Comment (EricCousineau-TRI) [reply site/docs/guide.md]: Done. Removed this entire section (seemed redundant, but did not thoroughly check).

**Review by meisterT — APPROVED** [review]

File: site/docs/guide.md [hunk 2]
```diff
@@ -1078,146 +1078,4 @@ these messages.
...
-    ```
-    1|1|0|0|0||PHASE|Launch Bazel
-    1|2|0|6000000|0||PHASE|Initialize command
-    1|3|0|168963053|278111411||VFS_READLINK|/[...]
-    1|4|0|571055781|23495512||VFS_STAT|/[...]
-    1|5|0|869955040|0||PHASE|Load packages
-    [...]
-    ```
-
-If Bazel appears to be hung, you can hit <kbd>Ctrl-&#92;</kbd> or send
```
Comment (meisterT) [inline site/docs/guide.md:1217]: This and the following lines need to be kept (or moved to the referenced place).

Comment (EricCousineau-TRI) [reply site/docs/guide.md:1217]: Done. Moved it; couldn't easily fit it into the profiling flow of text, so made a new subsection.

**Review by meisterT — APPROVED** [review]

**Pull request closed without merging**